_includes/manuals/1.12/5.8.3_secret_token.md

```diff
@@ -0,0 +1,8 @@
+
+The last file required to make a Foreman cluster work is __$app_root/tmp/secret_token.rb__. This file should contain a secret token that is modeled after __$app_root/config/initializers/local\_secret\_token.rb__. __$app_root/tmp/secret_token.rb__ is used by __$app_root/config/initializers/secret_token.rb__ to sign cookies. Once you have created the new secret_token.rb, restart Foreman and clear Foreman's cache:
```

Comment:
I'm unsure that this file is necessary, it would only have an effect if the local_secret_token.rb initializer was missing. It might be best to only ensure that the local secret token initialiser is correctly generated and in place?